.travis.yml

```diff
@@ -5,13 +5,16 @@ jdk:
   
 addons:
  sonarcloud:
-  organization: "sayantam-github"
+  organization: "3pillar-engineering-github"
   token:
-   secure: "GJMaVRQxvAsnZfuOubaycp9faIMa8LVelkmzDorj0THHsazE2ASQvqDuIEC6oAk2dDxUnx8CBWy4SS2AKtmvFb8mpT8A6Ta3NtxAbnP1mAmvzS4mzYXtVfPykDnBKRMAup1iEbB/dI9N8tN3HpjGuNFsCSlsJeRC4Zit9cU/K93B6SbO8qfSjrb/EiUvxH64fhEhG5mAiB5kgS6ixWLwh3dQlGI6DFQxI/gUGDr1F6d7bKCn0zmgEw5053hEktfRzy2GKIIUDV/xusHJ0IyPZatK1wnDVICsEsJ1KbbbeLZWrTQMbgIkO+0ZSyJWPD8R9aBaUuNMu1TZM5Ou4i+4PmUiSrj8+fSpZ1Iycyx+rd15rrFtGeSL4nQQ9PYDumtDH6uH6if8OBekQNsnOzxLrjW0rQl7QZnIYRBH9AwUs5azN2j1sxOCDq79d0QvANwrEt15YWaenawO7Z5nklOuNdgWs7df2dgSxpjgD7LwN8n+ZpBVDi33Gox15cB0H2kgQHL/AaqFICjW/YQC0mXf/pB5acGYfBZsQQVaWiBkh2GRET4SyDgLqnGS9O/IYe3ZdNzIBqT2MSApdTvpJ8dR/p57at0/nGeBRkuj9c4SyvGNeOxc9ZknJIBglShprEYYh1pV0foG+2Zmyh4tTdY9Mdt2GcNhUp6yq8DbjJScXQc="
+   secure: "yuCJcG9aVn3JdCDmgQq5pEA5WtFuIwyQUlF/mXldK0GcvxPE0YG4EF831FGTd3/SmQm7ee1j15TiGZDenDATJz848itq2d3uDpbm8o39nXQSFvjRnBjnn/S5kJVxpwLQMaIbPbnm8cdfk1aZeXGrudbPDzEkJlL/KRXRibxXgIfIbEpuAFRySfWL7yXXkMk2QuEHNrq+5JtFBlKB5LPCoSI3Ogps4V0/f5SPMfFV/kUSgD1WcBVaVkc3FECprxRtrpK2uWOEwtL+142pWD1ZFiQ9WTeQeLyiOIhQ3bc1Y19AIOOv6A+5bojzrbf/7GSuH3Bi2yQX0jtaRw8XEKhxmAbIRBoNjgiofgYpCHhiDDgqqk0Xi0EwNeaRL2inrEwP4kyKm9xPlsxfyuxyEyG2rszkrw/lf0kOJIn0DXMmTJl7DluQFHsTdR1tJRLRRkUvVixjms6VICAoTPMeV08RBL0wziIgqR1YQAAiTBo5VmaSaOxyzFDB/edGL7/Wj2eBmFDrHdW8G+zK0t34xbAHETYotmUAWEvMdRqDebZP+t/cRBXxMnXGtslRb/wR1IPqqhb5e4/5hsHvWx7gHWobAtSE1h3Q2AgK7SUKRir+0kDDzih1/iOe2HimdUFDMql5E3ejQCK37Qt8xWy3eAC2heCQG5MWDw2HA1QoRao2IUk="
+  github_token:
+   secure: "xrSjBA31jfHdzDUK8m2uWIrWtRTAXkuASCbTZPjb3+0Gq9s/1k/jYJKTS9C+fRSE1PFFoY9yyoSsSbhBjlnFL39+smIQhyOgiW+C55/LbuladZxQXYwkw1awIaM+wlJ2xYef4uoChZgm37EDRI5CAYbQpNbkzd1L64GFTRKBk6ELrD6kMk37b84TuTcek2cPbCW/pHGCxhHBncxHGEz0kmAmaTWx3GY5tEoO71I+huLSH99vafroGzstVVB+nS+0RLK0W8eI/Y1yhYq0mQJ/CmqjN7aR2cln32fbzWPdwhy1aiGL4GaC94NgQimZ64u2odu0xzFuxMJdbBU1TcDkwKeoDm2pDwYMzfDQDKOsVyTYvztlcWqnB6yJew36dw+PqaV9J17dHCt8pRuP7XNO6vaSzhokkW7YWx0Ht9+hczX0m8wgT1JUdAojZtRSNvho6/uByhxqEmYop1cAw/vcRSyeycpwEVKq5IyHiSiq1bZT5bfBbEd6nstztWwsNFTyTAFzyz7qB+oku/vdFGwlypaZdLR+Ae5SchmECJ0blE6/KeN3kvVjb0EscDIwjZ8D5sNRK2xC6bSjehb9U2bRPmat7sZlmYJ/jjHCH+30UPOAFdcCpgl6jXvTI50Bop5/JWMYPER+j8o5ZeqdzJtOntPc+TJJtx+foOVwl+Y6L7A="
   branches:

```

Comment:
@sayantam - Travis looks for the branches listed here and run analysis on those branches only. You are correct that we have to list current branches for this to work. However, there is another way where we can provide regular expressions to include the branches rather than full branch names. I'll look into this.